docs/GettingStarted.md

```diff
@@ -199,35 +148,6 @@ The Rack middleware can be configured using the global configuration object:
 In the example above, we've activated the Distributed Tracing flag, please
 see [distributed tracing](#Distributed_Tracing) for more details.
 
-#### Configure the tracer
-
-To modify the default middleware configuration, you can use middleware options as follows:
-
-    # config.ru example
-    require 'ddtrace/contrib/rack/middlewares'
-
-    Datadog.tracer.configure(
-      enabled: true,
-      hostname: localhost,
-      port: 8126
-    )
-
-    use Datadog::Contrib::Rack::TraceMiddleware, default_service: 'rack-stack'
-
-    app = proc do |env|
-      [ 200, {'Content-Type' => 'text/plain'}, "OK" ]
-    end
-
-    run app
-
-Available settings are:
-
-* ``tracer`` (default: ``Datadog.tracer``): set the tracer to use. Usually you don't need to change that value
```

Comment:
Are these options no longer available?

Reply:
good catch, @p-lambert I think we should document these no? https://github.com/DataDog/dd-trace-rb/blob/master/lib/ddtrace/contrib/rack/patcher.rb#L8-L15